output/csv/output.go

```diff
@@ -114,6 +117,7 @@ func newOutput(params output.Params) (*Output, error) {
 		ignoredTags:  ignoredTags,
 		row:          make([]string, 3+len(resTags)+1),
 		saveInterval: saveInterval,
+		timeFormat:   timeFormat,
```

Comment:
This lines gives an error on compilation:

> output/csv/output.go:120:17: undefined: timeFormat

I guess it should be 
```suggestion
		timeFormat:   config.TimeFormat,
```

but I also notice that github actions isn't running tests which is more worrying. 

Can you rebase on top of the current master and I hope that this will fix it :crossed_fingers: